core/keyserver/keyserver.go

```diff
@@ -102,8 +103,7 @@ func (s *Server) getEntry(ctx context.Context, userID string, epoch int64) (*tpb
 
 	neighbors, err := s.tree.NeighborsAt(ctx, index[:], epoch)
 	if err != nil {
-		log.Printf("Cannot get neighbors list: %v", err)
```

Comment:
This error is returned by an unexported function `getEntry`. All exported functions, which are gRPC handlers return gRPC errors.